### PR TITLE
Add engadminlocal to etc/hosts instructions

### PR DESCRIPTION
**Pull request by monfresh**
**Why**: It's required to be able to run the MilMoveLocal app
according to the rest of the README instructions.

### CODE REVIEW

**Comment by monfresh** [comment]
I believe I ran into this after running `make deps`, which runs `client_deps`, which runs `scripts/check-hosts-file`, which looks for `engadminlocal`.

My guess is that it was easier to include all the apps as opposed to having app-specific makefiles and instructions?

**Comment by monfresh** [comment]
Or are you saying we can remove all references to `engadminlocal` from this repo?

**Comment by monfresh** [comment]
@tinyels Any more thoughts on this? Is the correct solution to remove all references to `engadminlocal` from this repo?

**Comment by chrisgilmerproj** [comment]
Honestly I hate the practice of listing out the contents of `scripts/check-hosts-file` in the `README.md` because the script is the source of truth, not the `README.md`. Can we just delete the section here?

Also, to answer @tinyels 's question, the reason `engadminlocal` was added was to make it easier for devs to pick up and use the `milmove_admin` which they'll have to do in the future. It also reserves that namespace so that anyone wanting to add another line to the `/etc/hosts` file isn't tempted to choose it.

File: README.md [hunk 1]
```diff
@@ -316,6 +317,7 @@ Check that the file looks correct with `cat /etc/hosts`:
   127.0.0.1   orderslocal
   127.0.0.1   adminlocal
   127.0.0.1   primelocal
+  127.0.0.1   engadminlocal
 ```
 
 You can also verify this by running `scripts/check-hosts-file`.
```
Comment (chrisgilmerproj) [inline README.md]: This is the only line that really matters. Let's not have folks open up `/etc/hosts` anymore. The script gives the correct instructions for updating when its run.

**Comment by monfresh** [comment]
@chrisgilmerproj I updated this PR to remove the "setup hosts" section in the README so that we only rely on the `check-hosts-file` script, which gets run as part of `make prereqs`.

**Pull request merged**